fastestimator/summary/system.py

```diff
@@ -130,18 +132,19 @@ def update_batch_idx(self) -> None:
         else:
             self.batch_idx += 1
 
-    def reset(self, summary_name: Optional[str] = None) -> None:
+    def reset(self, summary_name: Optional[str] = None, system_config: Optional[str] = None) -> None:
         """Reset the current `System` for a new round of training, including a new `Summary` object.
 
         Args:
             summary_name: The name of the experiment. The `Summary` object will store information iff name is not None.
+            system_config: A description of the initialization parameters defining the associated estimator.
```

Comment:
what's it for?

Reply:
It's for the eventual traceability trace to have access to the estimator summary text for printing